07_Bootstrap/praktikum/index.html

```diff
@@ -0,0 +1,164 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Task 7 - Bootstrap</title>
+    <link rel="stylesheet" href="./assets/css/style.css" />
+    <link rel="stylesheet" href="./assets/bootstrap/css/bootstrap.min.css" />
+  </head>
+  <body>
+    <section>
+      <nav class="navbar navbar-expand-md">
+        <div class="container">
+          <a class="navbar-brand fw-semibold" href="#"><span class="text-success">e</span>Startup</a>
+          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNavDropdown" aria-controls="navbarNavDropdown" aria-expanded="false" aria-label="Toggle navigation">
+            <span class="navbar-toggler-icon"></span>
+          </button>
+          <div class="collapse navbar-collapse" id="navbarNavDropdown">
+            <ul class="navbar-nav ms-auto">
+              <li class="nav-item">
+                <a class="nav-link" aria-current="page" href="#">Home</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#">About</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#">Features</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#">Screenshot</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#">Team</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#">Pricing</a>
+              </li>
+              <li class="nav-item dropdown">
+                <a class="nav-link dropdown-toggle" href="#" role="button" data-bs-toggle="dropdown" aria-expanded="false"> Drop Down</a>
+                <ul class="dropdown-menu">
+                  <li><a class="dropdown-item" href="#">Action</a></li>
+                  <li><a class="dropdown-item" href="#">Another action</a></li>
+                  <li><a class="dropdown-item" href="#">Something else here</a></li>
+                </ul>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="">Contact</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </nav>
+    </section>
+
+    <section id="about">
+      <div class="container text-center">
+        <h1 class="fw-lighter mt-5">Simple System Fordiscount</h1>
+        <p class="mb-5 fw-semibold">Integer Cursus Bibendum Augue Ac Cursus</p>
+
+        <div class="row gy-sm-5 gy-5 justify-content-center">
+          <div class="col-xl-3 col-lg-3 col-md-3 col-sm-12">
+            <img src="./assets/img/cloud.svg" alt="cloud" width="50px" />
+            <p class="fw-bold my-4">INTRODUCING WHATSAPP</p>
+            <p>Lorem ipsum is simply dummy text of the printing and typesetting industry</p>
+            <a href="#" class="fw-semibold">Read More</a>
+          </div>
```

Comment:
kalo pengen pake symantic html untuk box image bisa pake figure > img > figcaption

reference:
[HTML `<figcaption>` Tag - w3school](https://www.w3schools.com/tags/tag_figure.asp)
[`<figcaption>`: The Figure Caption element - mdn web docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figcaption)